lib/rack/cache/meta_store.rb

```diff
@@ -81,7 +81,11 @@ def store(request, response, entity_store)
       headers.delete 'Age'
 
       entries.unshift [stored_env, headers]
-      write key, entries
+      if request.env['rack-cache.use_native_ttl'] && response.fresh?
```

Comment:
> would this be better as an option when creating the store ?

It might, but I was trying to keep this PR about the bug fix — the `rack-cache.use_native_ttl` env field was already present in other parts of the code (writing to the [entity store](https://github.com/rtomayko/rack-cache/blob/master/lib/rack/cache/meta_store.rb#L60), just above in this same file).
It just wasn't used properly with metastore writes, and wasn't documented.

I'm happy to make this a store option, but as that would be a breaking change (some users have code that reliad on `use_native_ttl`), maybe it'd be a better fit in a further PR?